### ISSUE TITLE
uuidFromObject would throw a error in example

### ISSUE THREAD

**Issue by huang12zheng** [zino-app/graphql-flutter#308]
**Describe the bug**
when:
object['id'] is int,it will throw a error
where it is: 
```
graphql-flutter/examples/starwars/lib/client_provider.dart 
final String id = object['id'] as String;  // line 7
```
**Expected behavior**
final String id = "${object['id']}";  // line 7

**Comment by truongsinh** [comment]
Thanks for reporting, can you make a PR to fix this?

**Comment by huang12zheng** [comment]
Yes, but it takes some time to learn pull request

**Comment by eusdima** [comment]
wouldn't ```object['id'].toString()``` be a cleaner fix?

**Comment by huang12zheng** [comment]
I think they are just the same.
and "${object['id']}" is more simple.
@eusdima

**Comment by mainawycliffe** [comment]
I have to agree with @eusdima for clarity purpose, in this case, `object['id'].toString()` is cleaner and clearer since its the only thing being assigned to the variable. This `"${object['id']}"` makes sense only when it is part of a larger string.

**Comment by micimize** [comment]
fixed in #317
